### ISSUE TITLE
Analytics GET is broken

### ISSUE THREAD

**Issue by xumx** [VapiAI/server-sdk-typescript#4]
https://github.com/VapiAI/server-sdk-typescript/blame/eef0da1c2e4360407c4d60365fb2c68b65e123cd/src/api/resources/analytics/client/Client.ts#L35

This Analytics Get doesn't make any sense. The main analytics API request param is missing, the API request has no payload. Please revert the changed.

**Comment by swaingotnochill** [comment]
Yup. Even I am having issues with this. There's no proper example as well how to make query to it.

**Comment by fern-support** [comment]
This is now fixed as the API has updated it from a `GET` to a `POST` request
